### DISCUSSION TITLE
Error when running gatsby clean

### DISCUSSION THREAD

**Discussion by oshincollin** [gatsbyjs/gatsby#38632] [Help]
When I run `gatsby clean` I get this error:

` ERROR  UNKNOWN

There was an unhandled error and we could not retrieve more information. Please run the command with the --verbose flag again.

 ERROR  UNKNOWN

[Error: EPERM: operation not permitted, rmdir 'C:\Users\hicol\Projects\sandboxes\gatsby-diagnosis\my-gatsby-diagnosis\.cache\.parcel-cache'] {
  errno: -4048,
  code: 'EPERM',
  syscall: 'rmdir',
  path: 'C:\\Users\\hicol\\Projects\\sandboxes\\gatsby-diagnosis\\my-gatsby-diagnosis\\.cache\\.parcel-cache'
}
`

I've been trying to complete the [Gatsby Getting Started Tutorial](https://www.gatsbyjs.com/docs/tutorial/getting-started/), but whenever I get to part 6, the `gatsby clean` command begins generating the above error. It's usually after the part of the tutorial where I add a blog directory to the project root. 

I've replicated this 3 times. I've tried going back to prior commits with no luck. I think it is a windows issue as I've tried on my mac with no problems. 

I've found other articles regarding windows marking files as read-only but this did not resolve the issue. 

Any ideas what might be going on?

**Comment by Bam92** [comment]
Hey @oshincollin,

Can you try some solutions [here](https://github.com/gatsbyjs/gatsby/issues/25658) ?
